userguide/network.rst

```diff
@@ -541,11 +523,12 @@ Creating a Link Aggregation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 **Before** creating a link aggregation, double-check that no
```

Comment:
Oddly worded, and not sure about the bold, but still: "**Before** creating a link aggregation, make sure that all interfaces to be used in the lagg have not been manually configured in :menuselection:`Network --> Interfaces`.